setup.py

```diff
@@ -51,11 +51,14 @@
 
 setup_args['install_requires'] = install_requires = []
 with open('requirements.txt') as f:
-    for line in f.readlines():
-        req = line.strip()
-        if not req or req.startswith(('-e', '#')):
-            continue
-        install_requires.append(req)
+    try:
+        for line in f.readlines():
+            req = line.strip()
+            if not req or req.startswith(('-e', '#')):
+                continue
+            install_requires.append(req)
+    except:
```

Comment:
why do you need try except? The file should be always there

Reply:
In some situations the program is unable to read the file correctly, so the creation of the treatment, in case the error occurs

Reply:
but it needs requirement. Isn't it?

Reply:
Fair, but security is never too much

Reply:
but you can't install the plugin without requirements... this should raise exception instead